SwanCustomEnvironments/swancustomenvironments/templates/customenvs.html

```diff
@@ -159,12 +103,6 @@ <h2 id="modal-title" style="text-align: left;"></h2>
     var loader = document.getElementById('swan-loader');
     var back_button = document.getElementById('back-button');
 
-    // Modal elements
```

Comment:
Would it be possible somehow to land in the extension with a diff configuration that the previously used? I would imagine we protect about any of this from the hub extension, right?

Reply:
Yes we do protect. If the user requested a different configuration than the one that is being created / has been created, that configuration is discarded since they agreed in the dialog to proceed to the current session. So the configuration of the current session dominates and stays. If they want to use the second configuration, they should pick the option to terminate the current session in the dialog.